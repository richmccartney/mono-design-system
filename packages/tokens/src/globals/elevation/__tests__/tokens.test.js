const motionTokens = require('../tokens.json');

describe('elevation tokens', () => {
  test(': matches snapshot', () => {
    expect(motionTokens).toMatchSnapshot();
  });
});
