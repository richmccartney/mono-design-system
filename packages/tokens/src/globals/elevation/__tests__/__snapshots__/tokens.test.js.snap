// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`elevation tokens : matches snapshot 1`] = `
Object {
  "elevation": Object {
    "deep": Object {
      "value": "-999999",
    },
    "default": Object {
      "value": "1",
    },
    "dropdown": Object {
      "value": "600",
    },
    "header": Object {
      "value": "400",
    },
    "mask": Object {
      "value": "200",
    },
    "masked": Object {
      "value": "100",
    },
    "modal": Object {
      "value": "900",
    },
    "overlay": Object {
      "value": "700",
    },
    "popup": Object {
      "value": "950",
    },
    "spinner": Object {
      "value": "800",
    },
    "sticky": Object {
      "value": "300",
    },
    "toast": Object {
      "value": "500",
    },
  },
}
`;
